src/BlockArea.cpp

```diff
@@ -517,7 +517,7 @@ bool cBlockArea::Write(cForEachChunkProvider * a_ForEachChunkProvider, int a_Min
 		LOGWARNING("%s: MinBlockY less than zero, adjusting to zero", __FUNCTION__);
 		a_MinBlockY = 0;
 	}
-	else if (a_MinBlockY > cChunkDef::Height - m_Size.y)
+	else if (a_MinBlockY >= cChunkDef::Height - m_Size.y)
```

Comment:
Here I thought it was possible to call FastSetBlock with an invalid value but I can't reproduce it even when entering incorrect values with the debugger. I will revert this change also.